framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2530,4 +2540,28 @@ private void printGitProperties() {
             System.out.println("IOException while reading git.properties: " + e.getMessage());
         }
     }
+
+    /** Extract the version of the Checker Framework */
+    private String getCheckerVersion() {
+        String version = null;
+        Properties p = new Properties();
+        String RLS_FILE = "/docs/developer/release/release.properties";
+        p.putAll(getProperties(getClass(), RLS_FILE));
+        String startTag = p.getProperty("checkers.ver.0");
+        String endTag = p.getProperty("checkers.ver.1");
+        String XML_FILE = "/docs/examples/MavenExample/pom.xml";
+        try {
+            InputStream in = getClass().getResourceAsStream(XML_FILE);
+            BufferedReader reader = new BufferedReader(new InputStreamReader(in));
+            String line = null;
+            while ((line = reader.readLine()) != null && version == null) {
+                if (line.split(startTag).length > 1) {
+                    version = line.split(startTag)[1].split(endTag)[0];
+                }
+            }
+        } catch (IOException e) {
+            System.out.println("IOException while reading version information: " + e.getMessage());
```

Comment:
Is it a bug if this line is reached?  If so, it is better to fail than to print a line that users are likely to overlook.